files/systemd-sysusers-config/placeholders.conf

```diff
@@ -0,0 +1,18 @@
+# Reserved system users and groups for future use in Endless OS 7.
+#
+# This config defines system users and groups that have static IDs in Endless OS
+# 6, which may own files in /etc or /var of existing Endless OS systems, and so
+# the UIDs and GIDs need to be reserved in EOS7 too.
+#
+# System user config should be installed in the element for the corresponding
+# system service. The users below are special cases where we don't (yet) have
+# an element defined in EOS7.
+
+#Type Name       ID
+g     colord     112
+u!    colord     106:112
```

Comment:
colord comes from colord package https://salsa.debian.org/debian/colord/-/blob/debian/1.4.6-2.2/debian/rules?ref_type=tags#L16

Reply:
Right, I moved this into a separate `colord.conf` file to avoid forking the freedesktop-sdk colord element